cmd/draft/connect.go

```diff
@@ -96,41 +105,85 @@ func (cn *connectCmd) run(runningEnvironment string) (err error) {
 
 	var connectionMessage = "Your connection is still active.\n"
 
+	exportEnv := make(map[string]string)
+
 	// output all local ports first - easier to spot
 	for _, cc := range connection.ContainerConnections {
 		for _, t := range cc.Tunnels {
-			err = t.ForwardPort()
-			if err != nil {
+			if err = t.ForwardPort(); err != nil {
 				return err
 			}
-			m := fmt.Sprintf("Connect to %v:%v on localhost:%#v\n", cc.ContainerName, t.Remote, t.Local)
-
-			connectionMessage += m
-			fmt.Fprintf(cn.out, m)
-		}
-	}
-
-	for _, cc := range connection.ContainerConnections {
-		readCloser, err := connection.RequestLogStream(deployedApp.Namespace, cc.ContainerName, cn.logLines)
-		if err != nil {
-			return err
+			if export {
+				var (
+					application = strings.Replace(strings.ToUpper(deployedApp.Name), "-", "_", -1)
+					container   = strings.ToUpper(cc.ContainerName)
```

Comment:
container names contain hyphens as well. Can we replace those with underscores too?

```
><> draft connect --detach
export EXAMPLE_GO_GO-CRAZY_SERVICE_HOST=localhost
export EXAMPLE_GO_GO-CRAZY_SERVICE_PORT=40743
```